### PR TITLE
Admin app

### PR DESCRIPTION
**Pull request by ArinaSavchenko**

### CODE REVIEW

File: AdminApp/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# AdminApp
```
Comment (andrei-kruhliakou-itechart) [inline AdminApp/README.md:1]: Please add some meaningful description for **your** project.

File: Airline_Web_API/Airline_Web_API/Services/AirportService.cs [hunk 2]
```diff
@@ -40,5 +49,57 @@ public async Task<IEnumerable<AirportViewModel>> GetAirports(string value)
...
+            if (airport == null)
+            {
+                return new Response<string>
+                {
+                    Success = false,
+                    Message = "There is no such airport"
+                };
+            }
+
+            _context.Remove(airport);
```
Comment (andrei-kruhliakou-itechart) [inline Airline_Web_API/Airline_Web_API/Services/AirportService.cs:95]: What are you going to do with all the entities that depend on airport, e.g. flights, tickets, etc? Are you going to delete them as well? What are you going to show in user tickets history if it contains some deleted airports?

Comment (ArinaSavchenko) [reply Airline_Web_API/Airline_Web_API/Services/AirportService.cs:95]: It's not finished version. I'll add logical delete using field 'status'

Comment (ArinaSavchenko) [reply Airline_Web_API/Airline_Web_API/Services/AirportService.cs:95]: And I would prefer to add it in new PR as I'll add this property not only for airport

Comment (andrei-kruhliakou-itechart) [reply Airline_Web_API/Airline_Web_API/Services/AirportService.cs:95]: Ok, please create separate task for this.

**Pull request merged**